mod_modular_vanilla/hooks/skills/actives/knock_back.nut

```diff
@@ -0,0 +1,14 @@
+::ModularVanilla.MH.hook("scripts/skills/actives/knock_back", function(q) {
+	// Convert the vanilla method of "setting" certain fields to instead be incremental changes
+	q.onAfterUpdate = @() function( _properties )
+	{
+		if (_properties.IsProficientWithShieldSkills)
+		{
+			this.m.FatigueCostMult *= ::Const.Combat.WeaponSpecFatigueMult;
```

Comment:
Shield Expert does not provide any fatigue cost reduction in Vanilla

Reply:
This seems to have snuck in due to some copy pasting I did. Will fix.